packages/deferable-reboot/tests/test_deferable_reboot.lua

```diff
@@ -0,0 +1,69 @@
+local config = require "lime.config"
+local test_utils = require 'tests.utils'
+local defreboot = require 'deferable_reboot'
+
+local uci = nil
+
+describe('Deferable Reboot tests #defreboot', function()
+
+    it('test load config from lime.config default value', function()
+        defreboot.config()
+        assert.is.equal(defreboot.DEFAULT_REBOOT_UPTIME, defreboot.min_uptime)
+    end)
+
+    it('test load config from lime.config', function()
+        config.set('system', 'lime')
```

Comment:
maybe use its own config file?

Reply:
I thought about that, the convention is to use it's own config file?

Reply:
I also think it's better to keep the configuration file separated, so that the package will have a chance to be merged into OpenWrt repository becoming independent from LibreMesh.

Reply:
> I thought about that, the convention is to use it's own config file?

it looks like this package is generic enough to be upstreamed at any point in the future... 
There is no written convention AFAIK, but my rule of finger would be to put in lime things tied to the lime ecosystem, and outside of it those that are more generic.

Reply:
I think if a configuration is not relevant to communities it may be in its own configuration file. On the other hand, if it's a configuration that makes sense for customizing a libremesh network it should be included within the lime configuration.

Reply:
This is a tricky one. Though I agree, I wonder if it should be in both places (as we do with dhcp, network and others). This might be the case so we can have the chance to update it to an openwrt package...

Reply:
Yes, probably for upstreaming it would be best to use it's own config file and also to read from lime.config (but as lime.config will not exists then we will need to add conditional code if lime is not installed)  I don't know how relevant would be for a community to change this. 
Anyhow I am adding this support

Reply:
Maybe we can do the same as we are doing with the rest, for lime-config to write the /etc/config/deferreboot file based on the config on /etc/config/lime if it is needed...

Reply:
Hm. I don't see how doing this helps upstreaming. If we add code to the same package that has to know about lime.config to do that then I don't know it has to be like in other package lime lime-deferable-reboot.... To much hassle for this simple code? 

Maaaybe we can have an automatic way of configuring other config files from lime config based on the name of the option? Probably too obscure...

Reply:
The way I was proposing is the same way as it is being done with the rest of the config: for /etc/config/lime to be used by lime-config to generate the config files of /etc/config/deferable-reboot , and for deferable-reboot not to have to know the existence of lime or not.

would that work?

Reply:
Yes, for this we should be doing another package or where do you think that the code that writes this config should be? Not in lime-system as this will tie lime with deferable-reboot. I think something like lime-deferable-reboot and that somehow by a plugin system this config is read from lime and writen to that file. That is what I tried to say in my last comment. And probably too complex system for a tiny peace of code. 

I think I will revert my last commit and just use the lime.config way of configuring it and decide how to untie it when, if ever, we try to upstream it. what do you think?